test/exports.spec.js

```diff
@@ -1,15 +1,14 @@
-import * as chai from 'chai'
-import * as path from 'path';
-import * as glob from 'glob';
-import toCamelCase from 'camelcase';
-import interopRequire from 'interop-require';
-import * as decorators from '../';
+const chai = require('chai');
+const path = require('path');
+const glob = require('glob');
+const camelCase = require('camelCase');
+const interopRequire = require('interop-require');
+const decorators = require('core-decorators');
```

Comment:
What I was shooting for was that the exports.spec.js and test.spec.js files should not require any transpiler.   I think as long as we're running tests on a modern version of node.js, we could use ESM imports here, but I wasn't sure.   I can restore them if you want.   Please advise.